app/controllers/TaskController.scala

```diff
@@ -265,19 +265,17 @@ class TaskController @Inject() (val messagesApi: MessagesApi) extends Controller
     TaskService.findAssignableFor(user).futureBox.flatMap {
       case Full(assignment) =>
         NewRelic.recordResponseTimeMetric("Custom/TaskController/findAssignableFor", System.currentTimeMillis - s)
-        val s2 = System.currentTimeMillis
-        TimeLogger.logTimeF("task request", logger.trace(_))(OpenAssignmentService.remove(assignment)).flatMap {
-          removeResult =>
-          NewRelic.recordResponseTimeMetric("Custom/TaskController/removeOpenAssignment", System.currentTimeMillis - s2)
-          if (removeResult.n >= 1)
+        TimeLogger.logTimeF("task request", logger.trace(_))(OpenAssignmentService.take(assignment)).flatMap {
+          updateResult =>
+          if (updateResult.n >= 1)
```

Comment:
`update` only updates one document by default (unless `multi: true` is specified), so it does not really matter